components/style/properties.mako.rs

```diff
@@ -4829,3 +4852,12 @@ pub fn longhands_from_shorthand(shorthand: &str) -> Option<Vec<String>> {
         _ => None,
     }
 }
+
+fn compute_font_hash(font: &mut style_structs::Font) {
```

Comment:
True - perhaps a comment in the font code to update that hash function if that ever changes? Otherwise, looks good to me.